tests/virt_autotest/libvirt_routed_virtual_network.pm

```diff
@@ -35,6 +35,11 @@ use version_utils 'is_sle';
 sub run_test {
     my ($self) = @_;
 
+    #Ensure that there is enough available default pool size for virtual network test
+    my $POOL = $self->get_available_pool();
+    die "Available POOL=$POOL. The SUT needs at least 80G of POOL" if $POOL < 80;
```

Comment:
In here guests are already created. I would execute this where you create your guests.

Reply:
sure, had added the comments as your mentioned.

Reply:
DONE

Reply:
I think this should be in libvirt_virtual_network_init.pm.

Reply:
sure, had moved to libvirt_virtual_network_init.pm

Reply:
Thank you 😄